client/packages/inventory/src/Stocktake/DetailView/DetailView.tsx

```diff
@@ -17,31 +17,29 @@ import { Footer } from './Footer';
 import { AppBarButtons } from './AppBarButtons';
 import { SidePanel } from './SidePanel';
 import { StocktakeSummaryItem } from '../../types';
-
 import { StocktakeLineEdit } from './modal/StocktakeLineEdit';
 import { ContentArea } from './ContentArea';
 import { AppRoute } from '@openmsupply-client/config';
-import { StocktakeLineFragment, useStocktake } from '../api';
+import { StocktakeFragment, StocktakeLineFragment, useStocktake } from '../api';
 import { StocktakeLineErrorProvider } from '../context';
+import { isStocktakeDisabled } from '../../utils';
 
-export const DetailView: FC = () => {
-  const isDisabled = useStocktake.utils.isDisabled();
-  const { isOpen, entity, onOpen, onClose, mode } =
-    useEditModal<ItemRowFragment>();
-  const { data, isLoading } = useStocktake.document.get();
-  const navigate = useNavigate();
-  const t = useTranslation('inventory');
-  const { HighlightStyles } = useRowHighlight();
-
+const StocktakeTabs = ({
+  id,
+  isDisabled,
+  onOpen,
+}: {
+  id: string | undefined;
+  isDisabled: boolean;
+  onOpen: (item?: ItemRowFragment | null | undefined) => void;
+}) => {
   const onRowClick = useCallback(
     (item: StocktakeLineFragment | StocktakeSummaryItem) => {
       if (item.item) onOpen(item.item);
     },
     [onOpen]
   );
 
-  if (isLoading) return <DetailViewSkeleton hasGroupBy={true} hasHold={true} />;
```

Comment:
Is removing this why we're seeing the 'No Data' while loading?

Reply:
It's been moved up a little, rather than removed.
The 'No Data' is something else - I felt that I'd spent long enough on the issue, so didn't spend time debugging that one, will have another look now that the PR is out. it's bothering me.